src/modules/site-v2/templates/strain/landing_page.html

```diff
@@ -2,12 +2,60 @@
 
 {% block content %}
 
+<style>
+    h2, h3 {
+        text-align: center;
+        color: black;
+        font-weight: bold;
+    }
+    .card a {
+        text-decoration: none;
+        color: black
+    }
+    .container {
+        display: flex;
+        justify-content: center;
+    }
+    .card {
+        text-align: center;
+        width: 25rem;
+        margin: 2rem
+    }
+    @media only screen and (max-width: 770px) {
+        .container {
+            flex-direction: column;
+        }
+        .card {
+            width:auto
+        }
+    }
+</style>
 
-<div>
-    <h2>Request Strains Landing Page</h2>
+<div class="container-fluid">
+    <h1>Request Strains</h1>
+    <h2>Please choose your affiliation:</h2>
+    <div class="container">
+        <div class="card shadow cardBorderBottom">
```

Comment:
Would recommend adding some padding to make these cards look a little bit more consistent with cards used elsewhere on the site. 

```
<div class="card py-5 shadow cardBorderBottom">
     ...
</div>
```
- `py-5` sets padding only on the top and bottom of the container